src/components/charlist/Charlist.tsx

```diff
@@ -2,15 +2,49 @@ import { Grid, Typography } from '@mui/material';
 import ParAvatar from 'components/styled/ParAvatar';
 import ParBox from 'components/styled/ParBox';
 import ParContainer from 'components/styled/ParContainer';
-import { FC } from 'react';
+import { FC, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
+import { Abilities } from './Abilities';
 import LiquidBar from './LiquidBar';
 import RollDiceButton from './RollDiceButton';
 
+export interface Ability {
+  enabled: boolean;
+  value: number;
+  override: number;
+}
+
 const Charlist: FC = () => {
   const { t } = useTranslation('translation', { keyPrefix: 'charlist' });
 
+  const [character, setCharacter] = useState<{ [x: string]: any }>({
+    abilities: {
+      acrobatics: { enabled: false, value: 0, override: 0 },
+      animalHandling: { enabled: false, value: 0, override: 0 },
+      arcana: { enabled: false, value: 0, override: 0 },
+      athletics: { enabled: false, value: 0, override: 0 },
+      deception: { enabled: false, value: 0, override: 0 },
+      history: { enabled: false, value: 0, override: 0 },
+      insight: { enabled: false, value: 0, override: 0 },
+      intimidation: { enabled: false, value: 0, override: 0 },
+      investigation: { enabled: false, value: 0, override: 0 },
+      medicine: { enabled: false, value: 0, override: 0 },
+      nature: { enabled: false, value: 0, override: 0 },
+      perception: { enabled: false, value: 0, override: 0 },
+      perfomance: { enabled: false, value: 0, override: 0 },
+      persuasion: { enabled: false, value: 0, override: 0 },
+      religion: { enabled: false, value: 0, override: 0 },
+      sleightOfHand: { enabled: false, value: 0, override: 0 },
+      stealth: { enabled: false, value: 0, override: 0 },
+      survival: { enabled: false, value: 0, override: 0 },
+    },
```

Comment:
This long thing can be moved to a separate file, or take out the above component